packages/error-reporting/README.md

```diff
@@ -44,6 +44,13 @@ var errors = require('@google-cloud/error-reporting')({
 errors.report(new Error('Something broke!'));
 ```
 
+- *One may even catch and report application-wide uncaught errors:*
+
+```js
+var errors = require('@google-cloud/error-reporting')();
+process.on('uncaughtException', e => errors.report(e));
```

Comment:
Actually somewhere in the README, can we add a section on uncaught exception handling that links to some Node.js docs? Too many gotchas here. For example, some people may be surprised that the application will no longer terminate.